update.sh

```diff
@@ -3,13 +3,15 @@ set -e
 
 declare -A gpgKeys
 gpgKeys=(
+	[7.0]='1A4E8B7277C42E53DBA9C7B9BCAA30EA9C0D5763'
```

Comment:
I would be OK with changing this file to only skip with a warning the "update version" portion and still regenerate `fpm` and `apache` for unknown versions, though.